elixir/apps/web/lib/web/router.ex

```diff
@@ -87,11 +91,13 @@ defmodule Web.Router do
   end
 
   scope "/:account_id_or_slug", Web do
-    pipe_through [:browser, :account]
+    pipe_through [:browser, :account, :client_auth]
 
-    live_session :client_redirect, on_mount: [Web.Sandbox, {Web.Auth, :mount_account}] do
-      live "/sign_in/success", SignIn.Success
-    end
+    get "/sign_in/deep_link", SignInController, :deeplink
```

Comment:
WDYT? Naming is always so hard, I don't have strong opinion on this one but feels like `deep_link` is too generic
```suggestion
    get "/sign_in/client_redirect", SignInController, :client_redirect
```

Reply:
I don't have a strong opinion here either.  I think `deep_link` is actually less generic, because it implies that the client will be given a deep link at some point.  However, now that you've mentioned it, I kind of think I like something more generic (at least to me) like `client_redirect`, because it gives flexibility if we would need to do something other than a deep link for some reason.  I'll change this to what you've recommended.

Reply:
Updated.